cfme/services/myservice.py

```diff
@@ -89,11 +161,16 @@ def wait_for_vm_retire(self):
         Returns: A string representing the VM state
         """
         def get_vm_details():
-            power_state = InfoBlock.text('Power Management', 'Power State')
+            power_details = SummaryTable(title='Power Management')
+            # power_state = SummaryFormItem('Power Management', 'Power State')
+            power_state = power_details.get_text_of('Power State')
+
             logger.debug('Service VM power state: {}'.format(power_state))
             if power_state == 'unknown':
                 # The VM power state is unknown, check lifecycle instead of power
-                retire_state = InfoBlock.text('Lifecycle', 'Retirement state')
+                retire_details = SummaryTable(title='Lifecycle')
```

Comment:
Removed this part as retired services will now show under "Retired services" section . Will keep a check for services moved to the retired section in next PR.